cmd/generate/runner.go

```diff
@@ -82,11 +82,22 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		ExtraLabels: nil,
 	}
 
+	if r.flag.Raw {
+		config, secret, err := gen.GenerateRaw(ctx, in)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		fmt.Println("---")
+		fmt.Printf(string(config) + "\n")
+		fmt.Println("---")
+		fmt.Printf(string(secret) + "\n")
+		return nil
```

Comment:
Can you pass the output directly to `helm install` call? It would be good to describe in the readme how to use that IMO.

Reply:
You have to redirect it to a file, and then use `-f values.yaml` flag in `helm install/template`.